OC Share Sheet/ShareViewController.swift

```diff
@@ -123,7 +123,7 @@ fileprivate func > <T : Comparable>(lhs: T?, rhs: T?) -> Bool {
     
     func createCustomInterface(){
         
-        let rightBarButton = UIBarButtonItem (title:NSLocalizedString("upload_label", comment: ""), style: .plain, target: self, action:#selector(ShareViewController.sendTheFilesToOwnCloud))
+        let rightBarButton = UIBarButtonItem (title:NSLocalizedString("upload_label", comment: ""), style: .plain, target: self, action:#selector(ShareViewController.uploadButtonTapped))
```

Comment:
In Xcode appears perfect, in one line it's more clear. Anyway we have to think if we want to do that.